scripts/flow/react-native-host-hooks.js

```diff
@@ -166,30 +175,30 @@ declare var nativeFabricUIManager: {
   registerEventHandler: (
     callback: (
       eventTarget: null | Object,
-      type: RNTopLevelEventType,
+      type: __RNTopLevelEventType,
       payload: Object,
     ) => void,
   ) => void,
 
   dispatchCommand: (node: Object, command: string, args: Array<any>) => void,
   sendAccessibilityEvent: (node: Object, eventTypeName: string) => void,
 
-  measure: (node: Node, callback: MeasureOnSuccessCallback) => void,
+  measure: (node: Node, callback: __MeasureOnSuccessCallback) => void,
   measureInWindow: (
     node: Node,
-    callback: MeasureInWindowOnSuccessCallback,
+    callback: __MeasureInWindowOnSuccessCallback,
   ) => void,
   measureLayout: (
     node: Node,
     relativeNode: Node,
     onFail: () => void,
-    onSuccess: MeasureLayoutOnSuccessCallback,
+    onSuccess: __MeasureLayoutOnSuccessCallback,
   ) => void,
   findNodeAtPoint: (
     node: Node,
     locationX: number,
     locationY: number,
-    callback: (Fiber) => void,
+    callback: (__Fiber) => void,
```

Comment:
You can type this as `Object`. It's typed as `Object` elsewhere in this file. E.g. `dispatchCommand`. Conceptually this is not a Fiber but an opaque reference.